crates/dojo-types/src/schema.rs

```diff
@@ -56,7 +56,13 @@ impl Ty {
             Ty::Struct(s) => s.name.clone(),
             Ty::Enum(e) => e.name.clone(),
             Ty::Tuple(tys) => format!("({})", tys.iter().map(|ty| ty.name()).join(", ")),
-            Ty::Array(ty) => format!("Array<{}>", ty[0].name()),
+            Ty::Array(ty) => {
+                if let Some(inner) = ty.first() {
+                    format!("Array<{}>", inner.name())
+                } else {
+                    "Array".to_string()
```

Comment:
How can we have an array without inner type?

Reply:
If the array is empty, we cannot get the inner type